pmlc/dialect/pxa/transforms/stencil.cc

```diff
@@ -43,12 +47,32 @@ class Stencil {
   // Target tensors strides, the first two are load, the third is aggregate
   llvm::SmallVector<mlir::StrideInfo, kNumTensors> tensorsStrides;
 
+  // Set of the op's BlockArguments.
+  BlockArgumentSet opBlockArguments;
+
+  // Index in tensors
+  BlockArgumentSet tensorIdxs[kNumTensors];
+  // Stride one index for the tensors
+  BlockArgumentSet strideOne[kNumTensors];
+  // The indeces used by the output tensor
+  BlockArgumentSet outIdxs;
+  // The accumulation indeces
```

Comment:
typo: indices (this same typo appears in many places in this file)

Reply:
Yep! Find/replace with a typo... :)

Reply:
Fixed.